.github/workflows/ci-nix.yml

```diff
@@ -290,7 +288,7 @@ jobs:
           - pkg: gateway-pkgs
             bin: gateway-cli
 
-    runs-on: buildjet-4vcpu-ubuntu-2004
+    runs-on: ubuntu-20.04
```

Comment:
I will change this before marking this PR ready, but I don't have this image in my repo's CI